cmd/interchaincmd/signatureaggregatorcmd/signatureAggregator.go

```diff
@@ -22,5 +22,6 @@ and configuring ICM signature aggregator.`,
 	app = injectedApp
 	cmd.AddCommand(newStopCmd())
 	cmd.AddCommand(newListCmd())
+	cmd.AddCommand(newStartCmd())
```

Comment:
can you make this fix to point to https://github.com/ava-labs/avalanche-cli/pull/2829 would like E2E + new commands to be merged at last